src/OrbitGl/AccessibleThreadBar.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "AccessibleThreadBar.h"
+
+#include "GlCanvas.h"
+#include "OrbitBase/Logging.h"
+#include "ThreadBar.h"
+#include "Track.h"
+
+namespace orbit_gl {
+
+int orbit_gl::AccessibleThreadBar::AccessibleChildCount() const { return 0; }
+
+const orbit_accessibility::AccessibleInterface* AccessibleThreadBar::AccessibleParent() const {
+  CHECK(thread_bar_ != nullptr);
+
+  if (thread_bar_->GetParent() == nullptr) return nullptr;
+  if (thread_bar_->GetParent()->GetAccessibleInterface() == nullptr) return nullptr;
+
+  // This relies on knowledge about AccessibleTrack: Accessibility is split into "virtual" controls
+  // that only exist as children in the accessibility tree. In particular, the track is split into
+  // a tab and a content area, and the thread bar is parented underneath the content area (child 1).
+  // This could be cleaned up by introducing "real" control elements for tab and content area
+  return thread_bar_->GetParent()->GetAccessibleInterface()->AccessibleChild(1);
+}
+
+std::string AccessibleThreadBar::AccessibleName() const { return thread_bar_->GetName(); }
+
+orbit_accessibility::AccessibilityRect orbit_gl::AccessibleThreadBar::AccessibleLocalRect() const {
+  CHECK(thread_bar_ != nullptr);
+
+  Vec2 pos = thread_bar_->GetPos();
+
+  // TODO(b/177350599): This could be cleaned up with clearer coordinate systems
+  if (thread_bar_->GetParent() != nullptr) {
```

Comment:
Yes, kind of... I think the core issue is that we're specifying absolute positions when it would make more sense to specify positions relative to the parent, then `pos` would be correct in both cases.

This can't actually happen since we're parenting all elements under `TimeGraph` currently, I just wanted to make sure this case works and is well-defined to have it unit-testable (when tracks are finally testable).